code/datums/dna.dm

```diff
@@ -272,6 +274,24 @@
 	return
 
 /////////////////////////// DNA MOB-PROCS //////////////////////
+/datum/dna/proc/update_body_size()
+	if(!holder)
+		return
+	var/desired_size
+	var/list/feat2num = list("Normal" = BODY_SIZE_NORMAL, "Tall" = BODY_SIZE_TALL, "Short" = BODY_SIZE_SHORT)
+	if(!species.default_features["body_size"])
+		desired_size = BODY_SIZE_NORMAL
+	else
+		desired_size ||= feat2num[features["body_size"]]
```

Comment:
this if else chain isn't needed, you already have normal bodysize in the list. Just read the list
```suggestion
	desired_size ||= feat2num[features["body_size"]]
```